### ISSUE TITLE
Notifier.show looses focus on Textfield in window

### ISSUE THREAD

**Issue by yozef** [caffeinalab/ti.notifications#22]
I wonder if that has been experienced by other developers, but imagine a window with text fields.
When a user has focus on a textfield, on return, a Notifier is shown, after the animation finishes, the keyboard looses focus on the textfield.

**Comment by kopiro** [comment]
Hi, can you provide a code example to test ?

**Comment by kopiro** [comment]
The standard iOS behaviour is that on return the keyboard lose focus anyway, not?

http://cl.ly/3D1S1W281n2V

**Comment by yozef** [comment]
You are right. I'll provide a small sample. I even might have a solution in mind.

**Comment by yozef** [comment]
Forget it was my stupid mistake. Closing.

**Comment by yozef** [comment]
When the Notification view closes (slides back up) the keyboard looses focus.

.xml:
~~~
<TextField id="tLastName" onReturn="go"/>
~~~

.js:
~~~
function go() {
       var obj = {message: msg};
	if (icon)
		obj.icon = icon;
	else
		obj.icon = '';
	if (OS_ANDROID)
		obj.view = $.vLogin;
	Alloy.Globals.Notifier.show(obj);
}
~~~

Result: After the Notifier shows the Notification, it slides back up & the keyboard looses focus even if the textfield gets tapped on after it first displays the notification. I'm guessing the FirstResponder gets allocated to the view that slides up rather than the textfield.
